### ISSUE TITLE
Can you improve  1.12.2, add enchantments for 1.12.2

### ISSUE THREAD

**Issue by hqddcxgw** [Derec-Mods/Mace-Mod#10]
Minecraft Version:1.12.2
Mod Version:
Can you improve  1.12.2, add enchantments for 1.12.2
Thanks for submitting an issue and I hope you have an awesome day!

**Comment by DereC4** [comment]
Enchants are in progress a bit busy due to summer courses atm

**Comment by ralseilii** [comment]
Yes im also hoping for enchants in 1.12.2 version , it will add fun to modpacks making
